README.md

```diff
@@ -16,11 +16,80 @@ No one likes monitoring/observability, everybody wants to deploy applications bu
 
 Releasing docker images at: [Quay](https://quay.io/repository/influxdb/telegraf-operator?tag=latest&tab=tags)
 
-## Adding telegraf-operator in development mode
+## Installing telegraf-operator in your Kubernetes cluster
+
+### Helm chart
+
+An up to date version of `telegraf-operator` can be installed by using the [InfluxData Helm Repository](https://helm.influxdata.com/).
+
+Simply run:
+
+```shell
+helm repo add influxdata https://helm.influxdata.com/
+helm upgrade --install telegraf-operator influxdata/telegraf-operator
+```
+
+To change one or more settings, please use the `--set` option - such as:
+
+```shell
+helm upgrade --install telegraf-operator influxdata/telegraf-operator \
+  --set certManager.enable=true
+```
+
+The `certManager.enable` setting will use [`cert-manager`](https://cert-manager.io/) CRDs to generate TLS certificates for the webhook admission controller used by `telegraf-operator`. Please note that this requires [`cert-manager`](https://cert-manager.io/) to be installed in the cluster to work.
+
+It is recommended to use a [values file](https://helm.sh/docs/chart_template_guide/values_files/) instead of setting name-values.
+
+It's also recommended to configure the [`classes.data`](https://github.com/influxdata/helm-charts/blob/62b039f/charts/telegraf-operator/values.yaml#L10-L18) values, which specify the telegraf-operator classes and how gathered data should be stored or persisted. For example:
```

Comment:
It might be worth linking to `## Global configuration - classes` lower down in the README

Reply:
Good point. Added.